src/concat/requireify/index-files.js

```diff
@@ -0,0 +1,32 @@
+var _ = require('underscore');
+
+function index(files) {

```

Comment:
This takes the list of files and assigns an index to each one. These are use to map the require calls within a bundle to different files.

For example:
##### `foo.js`
```javascript
module.exports = 2;
```

##### `bar.js`
```javascript
var foo = require('./foo');

module.exports = foo * 2;
```

In the compiled output, you'll see something like this:
```javascript
{
    1: [function(require,module,exports){
            module.exports = 2;
        }, {}],
    2: [function(require,module,exports){
            var foo = require('./foo');

            module.exports = foo * 2;
        },{'./foo':1}]
}
```

The keys on that object are the indices, and you can see in the second one it has a mapping from `./foo` to `1`.